test/staking/Integration.ts

```diff
@@ -514,7 +537,7 @@ describe("Integration", function () {
       const totalExpectedRewardToBeDistributed = REWARD_PER_EPOCH.mul(supportedNetworksWeights[0]).div(100);
 
       expect(totalPondDistributed).lt(totalExpectedRewardToBeDistributed);
-      expect(totalExpectedRewardToBeDistributed).is.closeTo(totalPondDistributed, 10);
+      expect(totalExpectedRewardToBeDistributed).is.closeTo(totalPondDistributed, totalPondDistributed.div(10000));
```

Comment:
The common question across all tests is if the error is as high as 0.1%

Reply:
max tickets are 65536   .......  ( minus 1) 
The error rate will be in the multiples of `1/65536`

when tickets are distributed equally i,e `[13072, 13072, 13072, 13072, 13073] (one of them will be ±1)`
the error rate increases to `1/13072`

Hence, in after running the tests multiple times, `1/10000` error rate was a chosen